### PR TITLE
Release notes for XTC 83

### PR DESCRIPTION
**Pull request by jowerner**

### CODE REVIEW

File: content/en/xtc/xtc-release-notes/v83.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+## Platform
+
+* Minor improvements and fixes that are not user-facing.
+
+
+## Load Testing
+
+* XTC caches the Maven repository for a load testing project to speed up the build time of your test suite. If the cached repository archive was corrupted, subsequent builds using that archive might fail. In these cases, the build will now fetch the Maven repository anew.
```
Comment (h-arlt) [inline content/en/xtc/xtc-release-notes/v83.md:24]:
```suggestion
* XTC caches the local Maven repository for a load testing project to speed up the build time of your test suite. However, if the cached repository was corrupted somehow, subsequent builds were likely to fail. Whenever XTC detects an integrity issue for the cached repository, the build will now fetch all required dependencies anew and update the cache such that subsequent builds can profit from it again.
```

File: content/en/xtc/xtc-release-notes/v83.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* Currently available XLT execution environments:
+    * XLT 7.x → [7.3.0]({{< relref "/xlt/release-notes/7_3_x" >}}) (deprecated, will be **retired on June 11, 2024**)
```
Comment (h-arlt) [inline content/en/xtc/xtc-release-notes/v83.md:26]:
```suggestion
    * XLT 7.x → [7.3.0]({{< relref "/xlt/release-notes/7_3_x" >}}) (deprecated, will be **retired on June 11, 2024**) - Please migrate your testsuite as soon as possible if you are still using it.
```

**Review by h-arlt — APPROVED** [review]
As usual, just a few suggestions. Feel free to apply or drop them.

File: content/en/xtc/xtc-release-notes/v83.md (hunk 2 above)
Comment (jowerner) [reply content/en/xtc/xtc-release-notes/v83.md]: Done.

**Pull request merged**